SPECS/golang/golang.signatures.json

```diff
@@ -3,7 +3,7 @@
     "go.20230802.5.src.tar.gz": "56b9e0e0c3c13ca95d5efa6de4e7d49a9d190eca77919beff99d33cd3fa74e95",
     "go.20240206.2.src.tar.gz": "7982e0011aa9ab95fd0530404060410af4ba57326d26818690f334fdcb6451cd",
     "go1.22.12-20250211.4.src.tar.gz": "e1cc3bff8fdf1f24843ffc9f0eaddfd344eb40fd9ca0d9ba2965165be519eeb7",
-    "go1.24.5-20250708.7.src.tar.gz": "c8dbbc78a39213d469cac915699a2581bd2d5d3f7f2f55394adedd365ecea46c",
+    "go1.25.1-20250903.7.src.tar.gz": "aeed32f132e54f5985164a0ea23a40000eadcb6650edbcd9f84248e30a88b372",
```

Comment:
The PR check build has failed due to this file missing in the blob store (or having a different hash). Please check, if you uploaded it.

Reply:
Thanks, due to the unusual circumstances this indeed hadn't happened.